src/pointer-analysis/dereference.cpp

```diff
@@ -2087,9 +2166,15 @@ void dereferencet::check_data_obj_access(
 
 void dereferencet::check_alignment(
   unsigned long minwidth,
-  const expr2tc &&offset,
+  const expr2tc &&offset_bits,
   const guardt &guard)
 {
+  // Perform conversion to bytes here until we figure out
+  // how to check alignment for bitfields.
+  minwidth = (minwidth + 7) / 8;
```

Comment:
Shall we explain the magic numbers (e.g., why 7 )?

Reply:
I've added an explanation now